lisa/tools/nvidiasmi.py

```diff
@@ -9,7 +9,7 @@
 class NvidiaSmi(Tool):
     # tuple of gpu device names and their device id pattern
     # e.g. Tesla GPU device has device id "47505500-0001-0000-3130-444531303244"
-    gpu_devices = (("Tesla", "47505500", 0), ("A100", "44450000", 6))
+    gpu_devices = (("Tesla", "47505500", 0), ("A100", "44450000", 6), ("H100", "44453233", 0))
```

Comment:
use below code, it will pass the CI check

```
    gpu_devices = (
        ("Tesla", "47505500", 0),
        ("A100", "44450000", 6),
        ("H100", "44453233", 0),
    )
```